docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -227,46 +222,75 @@ Accept: application/json
 
 ```json
 {
-  "name": "Header Images",
-  "template": "{{#each images}}<a href='{{image_url}}'><img src={{image_source}} style='width:33.3%'/></a>{{/each}}",
-  "schema": [
-    {
-      "type": "tab",
-      "label": "Content",
-      "sections": [
-        {
-          "label": "Product cards",
-          "settings": [
+   "name":"Header Images",
+   "template":"{{#each images}}<a href='{{link}}'><img src={{imageUrl.src}} style='width:33.3%'/></a>{{/each}}",
+   "schema":[
+      {
+         "type":"array",
+         "label":"Images",
+         "id":"images",
+         "defaultCount":3,
+         "entryLabel":"Image",
+         "thumbnail":"imageUrl.src",
+         "schema":[
             {
-              "type": "boolean",
-              "label": "Show product name",
-              "id": "showProductName",
-              "default": true
+               "type":"tab",
+               "label":"Content",
+               "sections":[
+                  {
+                     "settings":[
+                        {
+                           "type":"imageManager",
+                           "id":"imageUrl",
+                           "default":{
+                              "src":"https://cdn11.bigcommerce.com/s-n0i50vy/images/stencil/1280x1280/products/109/361/kinfolkessentialissue_1024x1024__22507.1456436715.jpg?c=2&imbypass=on",
+                              "type":"IMAGE_MANAGER"
+                           }
+                        },
+                        {
+                           "label":"Link",
+                           "type":"input",
+                           "id":"link",
+                           "default":"#"
+                        }
+                     ]
+                  }
+               ]
             }
-          ]
-        }
-      ]
-    }
-  ]
+         ]
+      }
+   ]
 }
 ```
 
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget-template/createwidgettemplate#requestrunner)
 
-For more information, see [Page Builder Overview](https://developer.bigcommerce.com/stencil-docs/page-builder/page-builder-overview).
+To learn more about Page Builder, see [Page Builder Overview](https://developer.bigcommerce.com/stencil-docs/page-builder/page-builder-overview).
 
 ## Resources
 
 ### Related endpoints
-* [Widgets API](/api-reference/storefront/widgets-api)
+* [Get All Categories](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategories)

```

Comment:
Is Get All Categories related to Widgets?

Reply:
This endpoint is not related to the Widgets API but it is useful when looking for a category ID. I will incorporate it into the tutorial itself.